### PR TITLE
chore:minor font changes;

### PR DESCRIPTION
**Pull request by just-s-i-d**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -73,10 +73,14 @@ input:focus {
     align-items: center;
     font-size: 20px;
     z-index: 10;
+    overflow: hidden;
 }
-header{
+
+
```
Comment (shelly19gkmit) [inline css/style.css]: remove extra one line

File: css/style.css [hunk 2]
```diff
@@ -368,6 +376,43 @@ button {
...
+    padding: 1%;
+    width: 80%;
+    margin: 5vh auto;
+}
+
+.testimonial-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
```
Comment (shelly19gkmit) [inline css/style.css:396]: remove this extra line

File: index.html [hunk 3]
```diff
@@ -33,6 +33,25 @@
...
+                            <path d="M4 12L20 12" stroke="#" stroke-width="2" stroke-linecap="round"></path>
+                            <path d="M4 6L20 6" stroke="#" stroke-width="2" stroke-linecap="round"></path>
+                        </g>
+                    </svg>
+                    <ul class="dropdown-links">
+                        <li><a href="#home">Home</a></li>
+                        <li><a href="#doctors">Doctors</a></li>
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="#contact-us">Contact</a></li>
+     -->
```
Comment (shelly19gkmit) [inline index.html]: Remove this extra code if not needed, if needed later, add a comment

File: css/style.css [hunk 4]
```diff
@@ -267,18 +272,21 @@ button {
     padding: 20px 10px;
     box-shadow: 0 0 10px 10px rgba(0, 0, 0, 0.1);
     width: 20%;
+    /* min-height: 20vh; */
```
Comment (shelly19gkmit) [inline css/style.css]: remove comments if not needed

File: css/style.css (hunk 1 above)
Comment (just-s-i-d) [reply css/style.css]: ok

File: css/style.css (hunk 4 above)
Comment (just-s-i-d) [reply css/style.css]: ok

**Comment by just-s-i-d** [comment]
ok i will do the required changes

**Pull request closed without merging**